compair/api/answer.py

```diff
@@ -249,6 +251,10 @@ def post(self, course_uuid, assignment_uuid):
                 abort(400, title="Answer Not Submitted",
                     message="You are currently not in any group for this course. Please contact your instructor to be added to a group.")
 
+        # set submission date if answer is being submitted for the first time
+        if not answer.draft and not answer.submission_date:
```

Comment:
This wasn't added to the update method?

Reply:
Sorry what update method? Isn't this the update method?

Reply:
This is the create answer endpoint (part of AnswerRootAPI) https://github.com/ubc/compair/blob/ab94c4aadb1bda028a357e045b6dd986a84566e2/compair/api/answer.py#L204

The answer update endpoint (part of AnswerIdAPI) can be found here
https://github.com/ubc/compair/blob/ab94c4aadb1bda028a357e045b6dd986a84566e2/compair/api/answer.py#L366